### PR TITLE
Refactor username->email

### PR DESCRIPTION
**Pull request by Erin-Mounts**
to reflect current Bridge API usage

### CODE REVIEW

**Comment by syoung-smallwisdom** [comment]
Can you keep the original method signatures and deprecate them, please?

**Comment by Erin-Mounts** [comment]
will do

**Comment by Erin-Mounts** [comment]
done did

File: BridgeSDK/BridgeAPI/SBBAuthManager.h [hunk 1]
```diff
@@ -104,13 +104,19 @@ extern  NSString * _Nonnull gSBBAppStudy;
  *
  *  @return The username, or nil.
  */
-- (nullable NSString *)usernameForAuthManager:(nullable id<SBBAuthManagerProtocol>)authManager;
+- (nullable NSString *)emailForAuthManager:(nullable id<SBBAuthManagerProtocol>)authManager;
+
+/*!
+ * For backward compatibility only. Implement emailForAuthManager: instead, which will always be called by the SDK in preference to this. 
+ */
+- (nullable NSString *)usernameForAuthManager:(nullable id<SBBAuthManagerProtocol>)authManager __deprecated;
```
Comment (syoung-smallwisdom) [inline BridgeSDK/BridgeAPI/SBBAuthManager.h]:
```` 
@Deprecated
 */
- (nullable NSString *)usernameForAuthManager:(nullable id<SBBAuthManagerProtocol>)authManager  __attribute__((deprecated("Please use emailForAuthManager: instead.")));
````

**Comment by syoung-smallwisdom** [comment]
looks good

**Pull request merged**